src/AmplifyMapLibreGeocoder.ts

```diff
@@ -1,4 +1,6 @@
 import { Geo } from "@aws-amplify/geo";
+import MaplibreGeocoder from "@maplibre/maplibre-gl-geocoder";
+import { createDefaultIcon } from "./createMarker";
```

Comment:
Would it work if we just load/import the geocoder's css here such that developers don't have to in their app?
`import "@maplibre/maplibre-gl-geocoder/dist/maplibre-gl-geocoder.css";`

Reply:
Will try it out and report back!

Reply:
It works!

Reply:
Awesome thanks, were you able to test both a react app and static html pages?

Reply:
I tested in a react app but not in a static html page

Reply:
We do need to test both scenarios. Last time there was a regression in static html pages.

Reply:
Yeah, as long as this works in the static pages, the code looks good to me.